### PR TITLE
Feat(influxdb): add influxdb backup support

### PR DESCRIPTION
**Pull request by angelfs**

### CODE REVIEW

File: tests/phpbu/Backup/Source/InfluxdumpTest.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        $runner->expects($this->once())
+            ->method('run')
+            ->willReturn($this->getRunnerResultMock(0, 'influxd'));
+
+        $target    = $this->createTargetMock();
+        $appResult = $this->getAppResultMock();
+        $appResult->expects($this->once())->method('debug');
+
+        $influxd = new Influxdump($runner);
+        $influxd->setup(['pathToInfluxDb' => PHPBU_TEST_BIN]);
```
Comment (sebastianfeldmann) [inline tests/phpbu/Backup/Source/InfluxdumpTest.php]: `pathToInfluxDb` should be `pathToInfluxdump`

File: tests/phpbu/Backup/Source/InfluxdumpTest.php [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        $runner->expects($this->once())
+            ->method('run')
+            ->willReturn($runnerResultMock);
+
+        $target    = $this->createTargetMock($file);
+        $appResult = $this->getAppResultMock();
+        $appResult->expects($this->once())->method('debug');
+
+        $influxd= new Influxdump($runner);
+        $influxd->setup(['pathToInfluxDb' => PHPBU_TEST_BIN]);
```
Comment (sebastianfeldmann) [inline tests/phpbu/Backup/Source/InfluxdumpTest.php]: `pathToInfluxDb` should be `pathToInfluxdump`

File: tests/phpbu/Backup/Source/InfluxdumpTest.php [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+     */
+    public function testSimulate()
+    {
+        $runner    = $this->getRunnerMock();
+        $target    = $this->createTargetMock();
+        $appResult = $this->getAppResultMock();
+        $appResult->expects($this->once())->method('debug');
+
+        $influxd = new Influxdump($runner);
+        $influxd->setup(['pathToInfluxDb' => PHPBU_TEST_BIN]);
```
Comment (sebastianfeldmann) [inline tests/phpbu/Backup/Source/InfluxdumpTest.php]: `pathToInfluxDb` should be `pathToInfluxdump`

**Comment by sebastianfeldmann** [comment]
Thanks for the implementation.

It looks great but I found some small issues that cause the unit tests to fail.
Would be great if you code fix those before I merge the PR :)

File: tests/phpbu/Backup/Source/InfluxdumpTest.php (hunk 1 above)
Comment (angelfs) [reply tests/phpbu/Backup/Source/InfluxdumpTest.php]: My bad, I solved this when I had running the test but I forgot to add in GIT...
Problem fix now.

**Pull request merged**